package.json

```diff
@@ -2,24 +2,28 @@
   "name": "gloss-translation",
   "version": "0.0.0",
   "license": "MIT",
-  "scripts": {},
+  "scripts": {
+    "postinstall": "nx run-many --target=prisma-generate --all"
```

Comment:
This script ensures that any packages that need a prisma client have one generated